clang/test/Analysis/Checkers/WebKit/ref-ptr-accessor.cpp

```diff
@@ -0,0 +1,86 @@
+// RUN: %clang_analyze_cc1 -analyzer-checker=alpha.webkit.UncountedCallArgsChecker -verify %s
+// expected-no-diagnostics
+
+template<typename T>
+class Ref {
```

Comment:
It looks like you're copying this definition in every pull request. Can you put it in the header and `#include` instead?

Reply:
Sure.